### DISCUSSION TITLE
Ban of the Russian region

### DISCUSSION THREAD

**Discussion by dmitry537** [WalletConnect/walletconnect-monorepo#3859] [Developer Support] [answered]
Hello! We use web3modal in our project. Everything is great and I am truly grateful to the developers and community for such tool. But I noticed that our users from Russia cannot use this component because api.web3modal.com is blocked for users from Russia.

It looks like the ideas of decentralization and web3 no longer have any meaning if we are facing such a problem even in the web3 world. 
I ask you to reconsider blocking the API for user from Russia, maybe your decision today will be different.

**Comment by glitch-txs** [comment, marked as answer]
Please read here
https://github.com/orgs/WalletConnect/discussions/3862
